plugins/jenkins/src/components/JobRunsTable/JobRunsTable.tsx

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Link, Table, TableColumn } from '@backstage/core-components';
+import { Box, IconButton, Tooltip, Typography } from '@material-ui/core';
+import { default as React } from 'react';
+import JenkinsLogo from './../../assets/JenkinsLogo.svg';
+import { useJobRuns } from './../useJobRuns';
+import { Job, JobBuild } from './../../api/JenkinsApi';
+import { JenkinsRunStatus } from './../BuildsPage/lib/Status';
+import VisibilityIcon from '@material-ui/icons/Visibility';
+import { jobRunsRouteRef } from '../../plugin';
+import { useRouteRefParams } from '@backstage/core-plugin-api';
+
+const generatedColumns: TableColumn[] = [
+  {
+    title: 'Number',
+    field: 'number',
+    render: (row: Partial<JobBuild>) => {
+      return (
+        <Box display="flex" alignItems="center">
+          <Typography paragraph>
+            <Link to={row.url ?? ''}>{row.number}</Link>
+          </Typography>
+        </Box>
+      );
+    },
+  },
+  {
+    title: 'Timestamp',
+    field: 'timestamp',
+    render: (row: Partial<JobBuild>) => {
+      return (
+        <Box display="flex" alignItems="center">
+          <Typography>
+            {row?.timestamp ? new Date(row?.timestamp).toLocaleString() : ' '}
+          </Typography>
+        </Box>
+      );
+    },
+  },
+  {
+    title: 'Result',
+    field: 'result',
+    render: (row: Partial<JobBuild>) => {
+      return (
+        <Box display="flex" alignItems="center">
+          {row.inProgress ? (
+            <Typography>In Progress</Typography>
+          ) : (
+            <JenkinsRunStatus status={row?.result} />
+          )}
+        </Box>
+      );
+    },
+  },
+  {
+    title: 'Duration',
+    field: 'duration',
+    render: (row: Partial<JobBuild>) => {
+      return (
+        <Box display="flex" alignItems="center">
+          <Typography>
+            {row?.duration
+              ? (row.duration / 1000).toFixed(1).toString().concat(' s')
+              : ''}
+          </Typography>
+        </Box>
+      );
+    },
+  },
+
+  {
+    title: 'Actions',
+    render: (row: Partial<JobBuild>) => {
+      const ActionWrapper = () => {
+        return (
+          <div style={{ width: '98px' }}>
+            {row?.url && (
+              <Tooltip title="View build">
+                <IconButton href={row.url} target="_blank">
```

Comment:
we avoid explicit blanks, `<Link component={IconButton} to={row.url}>` is better because it gets the consistent semantics of link behavior + any clicktracking / metrics / whatnot attached works as intended too

Reply:
done